src/main/java/com/connexta/store/adaptors/S3StorageAdaptor.java

```diff
@@ -110,18 +121,25 @@ public StorageAdaptorRetrieveResponse retrieve(@NotBlank final String key)
     }
   }
 
+  @Override
+  public void updateStatus(@NotBlank String key, @NotBlank String storeStatus) {
+    final Tag updatedTag = new Tag(STATUS_KEY, storeStatus);
```

Comment:
✏️ Shouldn't the STATUS_KEY constant be moved to this class because it's only used here?